app/views/admin/editions/edit.html.erb

```diff
@@ -20,15 +20,23 @@
       } %>
 
       <h2 class="govuk-heading-l">Your version</h2>
-      <%= render "legacy_form", edition: @edition %>
+      <% if [Publication, CaseStudy, CorporateInformationPage, DocumentCollection, NewsArticle, Speech, StatisticalDataSet, DetailedGuide].include?(@edition.class) %>
```

Comment:
I know this is bad and could be a method, but it will literally just be in place for the next 3 (small) PRs then will be removed